File: samples/Bad.java
// samples/Bad.java – intentional grammar errors
package samples;

public class Bad {

    // Missing return type
    public doSomething() {
        System.out.println("bad");
    }

    // Missing closing brace for method
    public void anotherMethod() {
        int x = 10;
        if (x > 5) {
            System.out.println("big")
        /* missing semicolon above, missing } for method */
    }

    public static void main(String[] args) {
        new Bad().doSomething();
    }
}

File: samples/bad.c
/* samples/bad.c – intentional grammar errors */
#include <stdio.h>

int missing_semicolon(void) {
    int x = 5
    return x;
}

void unclosed_brace(void) {
    if (1) {
        int y = 10;
    /* missing closing brace for function */
}

int main(void) {
    missing_semicolon();
    return 0;
}
